src/Common/Core/Impl/Extensions/StreamExtensions.cs

```diff
@@ -3,25 +3,26 @@
 
 using System;
 using System.IO;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.Common.Core {
     public static class StreamExtensions {
-        public static async Task CopyAndFlushAsync(this Stream source, Stream destination, IProgress<long> progress = null) {
-                await source.CopyToAsync(destination, progress);
+        public static async Task CopyAndFlushAsync(this Stream source, Stream destination, CancellationToken cancellationToken, IProgress<long> progress = null) {
```

Comment:
Usually in public methods it is a last parameter and has a default value `default(CancellationToken)`. Do we want to follow this pattern?